src/layouts/Header/NavBar/Navbar.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function NavBar() {
+  return <div></div>;
+}
+
+export default NavBar;
```

Comment:
You have NavLink component and NavLinks component. Why did you make this component? If it is wrapper just for navigation and logo, you can use div tag for wrapping this components or rename somehow differently this component. But maybe you can use just CSS properties for it.

Reply:
I'd like to see Nastya's idea moving forward, but I have a gut feeling that Gleb is right.

Reply:
I agree with you, removed unnecessary component, thanks